core-ui/src/components/Extensibility/components/Columns.js

```diff
@@ -1,13 +1,12 @@
 import React from 'react';
 
 import { Widget } from './Widget';
-import './Columns.scss';
 
 export function Columns({ structure, ...props }) {
   return (
-    <div className="extensibility-columns">
+    <div className="panel-grid">
       {structure.children.map(child => (
-        <Widget structure={child} {...props} />
+        <Widget structure={child} hasExternalMargin={false} {...props} />
```

Comment:
Maybe we can rename `hasExternalMargin` because for me this name suggests the opposite behavior.
`hasExternalMargin={false}` -> the external wrapper **don't have** margin so add one
`hasExternalMargin={true}` -> the external wrapper **have** margin so skip it

Reply:
I used the same as it is in `GenericList`, so we have to think about remaining everything